tasks/pocket_sand.py

```diff
@@ -0,0 +1,20 @@
+import random
+from discord_bot import STATIC_PATH
+import asyncio
+import os
+import discord
+from tasks.base import BaseTask
+
+
+class DaleAttack(BaseTask):
+    id = 'dale-attack'
+    channel = None
+
+    async def task(self):
+        if not self.channel:
+            self.channel = self.client.get_channel(596772352835190823)
```

Comment:
This should be converted to a .env variable and should have a check to see if the variable has been filled. If not, end the task in the before command.

Reply:
Agreed, though this calls out that we don't have that set up for the gru nose command: https://github.com/michaelhillebrand/huskiebot/blob/f805f00e18a6788b65eb9f631e3ba4addfea8e23/tasks/gru_nose.py#L17